README.md

```diff
@@ -0,0 +1,43 @@
+# Software Engineering Team Design Project
+
+## Getting started with development
+
+Clone the GitHub repository and create your own branch using the auto-generated ClickUp naming convention.
+
+### Making code changes
+
+Husky is set up at the root of the project.
+
+Husky will run Prettier before ever commit (pre-commit hook). This will ensure that we have consistent code style across all developers.
```

Comment:
I think there is a typo here: 'ever' -> every

Reply:
You are right